cmd/install/install.go

```diff
@@ -480,6 +489,46 @@ func hyperShiftOperatorManifests(opts Options) ([]crclient.Object, error) {
 		objects = append(objects, userCABundleCM)
 	}
 
+	if mgmtClusterCaps.Has(capabilities.CapabilityImage) && mgmtClusterCaps.Has(capabilities.CapabilityProxy) {
+		imageRegistryCABundle, err := getImageRegistryCABundle()
+		if err != nil {
+			log.Log.Error(err, "unable to get additional image registry CA bundle")
```

Comment:
I think we would want to just return the error here. I don't think we want logging here.

Reply:
Gotcha, modified to return the error here instead of logging!